apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/index.tsx

```diff
@@ -5,18 +5,44 @@ import { useCallback, useState } from 'react'
 import { ConversationMetadata } from '@latitude-data/compiler'
 import { DocumentVersion } from '@latitude-data/core/browser'
 import { useCurrentCommit } from '@latitude-data/web-ui'
+import { useDocumentParameters } from '$/hooks/useDocumentParameters'
 import {
   PlaygroundInput,
-  useDocumentParameters,
-} from '$/hooks/useDocumentParameters'
+  useDocumentParameters as useDocumentParametersOld,
+} from '$/hooks/useDocumentParameters/oldHook'
 import { useFeatureFlag } from '$/hooks/useFeatureFlag'
 
 import { Header } from '../Header'
 import Chat from './Chat'
 import { DocumentParams } from './DocumentParams'
-import { InputParams } from './DocumentParams/Input'
+import { OldInputs } from './OldInputs'
 import Preview from './Preview'
 
+// FIXME: Remove oldParameters when we remove feature flag
```

Comment:
why do we have old parameters? everyone is with the new ui no?

Reply:
Not yet